extensions/applicationinsights-dependencies-js/src/ajax.ts

```diff
@@ -271,7 +271,7 @@ export class AjaxMonitor extends BaseTelemetryPlugin implements IDependenciesPlu
     public static identifier: string = "AjaxDependencyPlugin";
 
     public identifier: string = AjaxMonitor.identifier;
-
+    public readonly config: IConfig;
```

Comment:
Lets not add this (unless we "really" need to) as this is now accessible via the `core.config` as they are all the same